e2e/src/test/groovy/io/kubernetes/client/e2e/kubectl/KubectlRolloutDaemonSetTest.groovy

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.e2e.kubectl
+
+import java.nio.charset.StandardCharsets
+
+import io.kubernetes.client.extended.kubectl.History
+import io.kubernetes.client.extended.kubectl.Kubectl
+import io.kubernetes.client.openapi.Configuration
+import io.kubernetes.client.openapi.models.V1DaemonSet
+import io.kubernetes.client.openapi.models.V1PodTemplateSpec
+import io.kubernetes.client.util.ClientBuilder
+import io.kubernetes.client.util.Streams
+import io.kubernetes.client.util.Yaml
+import spock.lang.Specification
+
+class KubectlRolloutDaemonSetTest extends Specification {
```

Comment:
Perhaps it would be better to name this class "KubectlRolloutTest"？

Reply:
sgtm, we can append future tests to this class for other types of workloads